install_files/ansible-base/roles/app-test/tasks/main.yml

```diff
@@ -14,3 +14,7 @@
 - include: dev_setup_xvfb_for_functional_tests.yml
 
 - include: disable_mprotect_ff_plugin_container.yml
+
+- include: download_verify_tbb.yml
+
+- include: download_ff_esr.yml
```

Comment:
... I think I may have outsmarted myself with trying to just get latest FF from apt.  I feel like we need FF 45 or 46 in order to use Selenium 2.53, as latest FF needs Geckodriver.  

We could keep the current FF 46, as we're now only using it as a fallback, or try to work out what Geckodriver needs?  But it may well need a more recent version of Selenium as well - which seems hard to have as well as 2.53.